Makefile

```diff
@@ -37,4 +37,34 @@ depends:
 	go get -d ./...
 
 build: depends
-	cd vflow; go build $(LDFLAGS) -o vflow $(GOFILES)
+	cd vflow; go build $(LDFLAGS) 
+
+build-windows: depends
+	cd vflow; gox $(LDFLAGS) 
```

Comment:
This something worth having as GOX is one of the many cross compile tools just happens to be the one I use.  Should that not be used?  Or something else?